tests/unit/mixins/Registry.ts

```diff
@@ -57,7 +57,7 @@ registerSuite({
 			class IntegrationTest extends TestWithRegistry {
 				render() {
 					return v('div', [
-						w('test', { id: `${Math.random()}` })
+						w<any>('test', { id: `${Math.random()}` })
```

Comment:
Yes if it's any more than the default (which is `WidgetProperties`), this was recommended (undocumented most likely) anyway previously as because otherwise you didn't get the type guards for the properties.